src/test/java/selenium/tutorial/studies/HandlingAlerts.java

```diff
@@ -0,0 +1,91 @@
+package selenium.tutorial.studies;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.htmlunit.HtmlUnitDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import com.sun.corba.se.impl.oa.poa.ActiveObjectMap.Key;
+
+import junit.framework.Assert;
+
+public class HandlingAlerts {
+	
+	WebDriver driver;
+	
+	@BeforeTest
+	public void setup() throws IOException {
+
+		InputStream input = HandlingAlerts.class.getClassLoader().getResourceAsStream("testData.properties");
+		Properties property = new Properties();
+		property.load(input);
+		
+		if("chrome".equals(property.getProperty("activeBrowser"))) {
+			driver=new ChromeDriver();
+			driver.manage().window().maximize();
```

Comment:
same code driver.manage().maximize() is being used in both places. can you declare it in a common place so that both browsers can execute it. u can reduce lines of codes

Reply:
can u add deleteAllCookies()

Reply:
done